Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in activejob-dispatch_rider.gemspec
 gemspec
+
+gem "dispatch-rider", github: "payrollhero/dispatch-rider", ref: "9120a4e14217104d65ca9ddd41af138bf66dfb93"
```

Comment:
Yup. That's why there's a note to not merge. This is away to include it in the gem before it's released. Unfortunately CircleCI is not able to load it so it only works locally.